### ISSUE TITLE
Queries on labels with spaces not working

### ISSUE THREAD

**Issue by dosumis** [loopasam/Brain#13]
According to [this example](https://github.com/loopasam/Brain/wiki/Examples#wiki-labels) queries using getSubClassesFromLabel should work for labels with spaces  as long as those labels are quoted.  This does not appear to be the case.  A simple example:

```
    Brain pato = new Brain();
    pato.learn("http://purl.obolibrary.org/obo/pato.owl");

    List<String> colors = pato.getSubClassesFromLabel("color", false);  // This works
    for (String subClass : colors) {
        System.out.println("Subclass of color: " + subClass);
    }

    List<String> oqs = pato.getSubClassesFromLabel("'optical quality'", false);   // doesn't work
    for (String subClass : oqs) {
        System.out.println("Subclass of 'optical quality: " + subClass);
    }
}
```

**Comment by loopasam** [comment]
The quotation mark `'` has to be explicitly present in the label in order to be handled by the current version of Brain. If no quotation marks are saved in the original ontology, then the label won't be usable in the query.

E.g: The class `PATO:0001300` has as `rdfs:label` `optical quality`, without quotation marks in the original OWL file (`http://purl.obolibrary.org/obo/pato.owl`). Therefore no recognised.

The current implementation is arguably a good solution. Things to consider before making changes:

1) As far as I know, there is no strong specification for `rdfs:labels`, you can have as many as you like, with the same name, etc... which makes them really ambiguous when formulating queries. Therefore label-based methods should be just considered as features to facilitate the interaction with the ontology - nothing else shamefully. Even if in practice, they are the best interface with the user.

2) Protege adds extra `'` to glue the text and disambiguate some. I don't really like adding extra information that doesn't exist in the ontology, as it can make things pretty tricky to debug and understand. If multiple labels are present or the same value used for multiple entities, it becomes impossible in Protege to know what is what too.

Morality: Labels are there to simplify, but no one should unfortunately rely on them, according to the specs (or my understanding of it). At the time being, if you want to refer to entities using labels in Brain, the label value should explicitly contain `'`.

**Comment by dosumis** [comment]
I agree that one shouldn't rely on labels in production code and that referring to entities by their labels is unsafe for ontologies without a unique name constraint or with multiple labels per term.  

However, a large number of OBO ontologies follow both these constraints and querying on IDs is very inconvenient for scripts taking using input of queries.  So, it would be great if you could allow querying on labels for ontologies that passed both of these constraint tests. 

 Understandable if this is too specialised a request for Brain though.
